webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/InterpolatedValueCompiler.scala

```diff
@@ -172,6 +177,7 @@ object PropertyParserTokens {
   final case class Param(path: List[String])       extends AnyVal with Interpolation
   //here, we keep the case as it is given
   final case class Property(path: List[String], opt: Option[PropertyOption]) extends Interpolation
+  final case class RudderEngine(engine: String, method: List[String], param:ConfigValue ) extends Interpolation
```

Comment:
If we choose `${engine.name[param1][param2] | option1 = xxx, option2 = xxx }` format should we add a `Option[List[String]]`